docs/management/ha.md

```diff
@@ -39,13 +39,51 @@ Here are the possible cases and how they are dealt with:
 * **Lost storage but not network**: if the host can contact a majority of pool members, it can stay alive. Indeed, in this scenario, there is no harm for the data (can't write to the VM disks). If the host is alone (i.e can't contact any other host or less than the majority), it goes for a reboot procedure.
 * **Lost network but not storage (worst case!)**: the host considers itself as problematic and starts a reboot procedure (hard power off and restart). This fencing procedure guarantees the sanity of your data.
 
+## Requirements
+
+Enabling HA in XCP-ng requires thorough planning and validation of several prerequisites:
+
+- **Pool-Level HA only**: HA can only be configured at the pool level, not across different pools.
+- **Minimum of 3 hosts recommended**: While HA can function with just 2 XCP-ng servers in a pool, we recommend using **at least 3** to prevent issues such as a split-brain scenario. With only 2 hosts, an equal split could result in one host being randomly fenced.
```

Comment:
My understanding of split brain is not that one host is randomly fenced, but that both hosts, when losing connection to the other, risk fencing (or worse, running the same VMs at the same time? I don't know if the latter can happen).